Intersections_3/include/CGAL/Intersections_3/internal/Triangle_3_Triangle_3_intersection.h

```diff
@@ -39,11 +39,15 @@ struct Point_on_triangle
 {
   // triangle points are not stored in this class but are expected
   // to always be passed in the same order. For a triangle pqr,
-  // edge 0 is pq, edge 1 qr and edge 2 rp. Point 0 is p, 1 is q and 2 is r.
+  // edge 1 is pq, edge 2 qr and edge 3 rp. Point -1 is p, -2 is q and -3 is r.
+  //
+  //  (0,-) vertex of t2 inside t1
+  //  (-,-) common point of t1 and t2
+  //  (-,0) vertex of t1 inside t2
+  //  (-,+) vertex of t1 on an edge of t2
+  //  (+,+) edge of t1 intersecting edge of t2
+  //  (+,-) vertex of t2 on an edge of t1
```

Comment:
Wouldn't it be  better to have these "magical" numbers as as elements of and `enum` or `enum class` ?